README.md

```diff
@@ -1,24 +1,121 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 Hello Rails Back End <a name="about-project"></a>
 
-* System dependencies
+**Hello Rails Back End** is a rails API for getting a random greeting.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
+Ruby on Rails, Visual Studio Code
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+- **Ruby on Rails**
+- **PostgreSQL**
 
-* How to run the test suite
+### Key Features <a name="key-features"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+- **Random Greetings**
+- **API Integration**
 
-* Deployment instructions
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* ...
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Setup <a name="setup"></a>
+
+To use this project, Clone this repository to your desired folder by pasting this command in your command line interface:
+
+  - `git clone https://github.com/binyamolango/hello-rails-back-end.git`
+  - `git clone https://github.com/binyamolango/hello-react-front-end.git`
+
```

Comment:
- You have done well so far, though I can see the link to your frontend here, but according to the project requirements you should have a link that leads directly to the frontend repo, please fix this 👍 👍